content/articles/secondary-dns-provider-easy-dns.markdown

```diff
@@ -0,0 +1,100 @@
+---
+title: Secondary DNS with EasyDNS
+excerpt:
```

Comment:
This could probably be the same as whatever we decide on with DNSMadeEasy, just with the provider name swapped out.